### ISSUE TITLE
Booting trouble

### ISSUE THREAD

**Issue by acetechy72** [dpitts/f16v4_issues#20]
Hello everyone.   Would appreciate some help from the community.
I have a Falcon F16V4.   I was having trouble with intermittent booting.   I had two times where the board would output 100% white only to the LEDs and the Serial ports would not work.  I could not reboot the board (trying to power down/up and finally default reset using the onboard method).   The OLED would also not respond.    The only item I could do was to leave the board alone for >24 hours.    Then finally I could reset it.    Last night, I tried to update the firmware to build 14.    I used the web interface method.   The firmware uploaded to the board, but hung up in the reboot process (showing bootloader on the OLED).   I left the board for more than 1 hour.   Finally, I tried to power down/up once again.   The board will power up but there is no display on the OLED.     I have also tired to do a firmware upgrade from a Micro SD card on the board.   The result is the same.   In both cases, I have lost network connectivity.   Can anyone help on trouble shooting?   I would appreciate it.

**Comment by keithsw1111** [comment]
First of all lets see if we can capture what is going on with the board right now.

1. Install a SD card
2. While holding down the left and right buttons power the board.
3. At the prompt press the up button and save the current controller configuration to the SD card as a .DAT file.
4. Send the .DAT file to support@pixelcontroller.com referencing ticket #20

Once done lets try resetting the board.

While holding down the select button power the board. This should erase all settings back to a fresh install so if there is anything corrupt it will be cleared out.

If that does not work we have the option of resetting everything back to just the boot loader but i would prefer to hold that back until we have tried other things.

It would also be worth checking you are powering the board correctly. Are you using 12V or 5V to power the board and which connector are you using to apply power.

If you would like some help best place to catch us is in xLights zoom where we often are but also numerous other people are who can help you isolate the problem. They will help you validate that you are doing everything correctly and can also guide you further on some of the processes above.

**Comment by acetechy72** [comment]
keithsw1111,
Thanks for the quick response!   Really appreciate it.
I was able to capture the dat file and email.  It may be an issue.  I tried to read the dat file and it seemed corrupted.  
Your support helped me to get the controller operational yesterday.  I found it interesting how I got it to work.
Once I ran the protocol to download the Dat file, the board went back to bootloader.   However, I tried to cycle the power and the OLED did not work once again.   After a little while checking, I found that if I remove the OLED for ~10seconds or so and then power the board, the OLED works.  I am not that familiar with the specifics of the OLED architecture, but is it possible it some cap properties?   or there I may have a quality issue on the OLED.   Thinking back it is possible that each issue I have had seems to be once the power is turned off to the controller and the potential OLED issue.    
Also, I did confirm the power.  I am using the V2 bank for power with the a voltage rating of 12.0V.  I monitored it for a while with a meter and confirmed the power fluctuates from 11.98 to 12.01 (if you believe the meter is that accurate) at the on state.  

I would appreciate your thoughts about the issue as I am thinking of ordering a replacement OLED.  
BTW - I do like the product, software and support.   Keep it up.   

Thanks again the show is up and running for the kiddos.

**Comment by keithsw1111** [comment]
The DAT file is not meant to be readable so that is ok.

I am not sure why removing the OLED would have made any difference. While the CPU talks to it it is very much a one way conversation.

The fact you are powering it with 12 Volts makes the voltage unlikely to be an issue.

I will take a look at the config file and see if i can reproduce the problem.

**Comment by keithsw1111** [comment]
Is this still an issue?

**Comment by keithsw1111** [comment]
Closing due to no response. If this is still an issue we can reopen.

**Comment by acetechy72** [comment]
Hello Keith,
I am sorry to miss your email.  It is a bit odd.  I was able to get the
display up and running once again but I did have the issue just after the
Christmas holiday.  I have broken down the display and am planning to do a
deeper investigation next month.  It is strange.  The fialure mode is all
lights on (various colors) and the display does not light up.
Thanks for your follow up.
Kevin
